pandas/core/groupby/groupby.py

```diff
@@ -3493,10 +3502,9 @@ def _get_cythonized_result(
         cython_dtype: np.dtype,
         numeric_only: bool = False,
         needs_counts: bool = False,
-        needs_nullable: bool = False,
-        needs_mask: bool = False,
         pre_processing=None,
         post_processing=None,
+        how: str = "any_all",
```

Comment:
Could you document this in this function?

Reply:
Thx, done